locale/zh-tw/get-involved/index.md

```diff
@@ -8,6 +8,11 @@ layout: contribute.hbs
 ## 社群討論
 
 * [GitHub issues 清單](https://github.com/nodejs/node/issues)是討論 Node.js 核心功能的好地方。
+* 想實时聊聊關於 Node.js 開發方面的內容，可以使用以下之一的平臺：
```

Comment:
's/實时/即時/g'

Reply:
@PeterDaveHello：

![image](https://user-images.githubusercontent.com/52018749/124253586-c4c95300-db5a-11eb-903d-3bb952187fd0.png)

According to the latest version of English, it seems it has "one of....". That's the reason why "之一" is here.

Reply:
> 

@PeterDaveHello：

![image](https://user-images.githubusercontent.com/52018749/124253586-c4c95300-db5a-11eb-903d-3bb952187fd0.png)

According to the latest version of English, it seems it has "one of....". That's the reason why "之一" is here.

Reply:
IMO `以下` explains enough and translation shouldn't be done words by words, the expression `下列其中之一` seems to be too long and doesn't seem to be a common usage in my life.

Reply:
> IMO `以下` explains enough and translation shouldn't be done words by words, the expression `下列其中之一` seems to be too long and doesn't seem to be a common usage in my personal experience.

I take what you mean, and sometimes it's not right to translate word by word and we can just understand what it means instead of misunderstanding. I wonder if I say "以下平台", will someone misunderstand by using them all? And in fact we just use one of them.....That's what I'm worried about.

Reply:
I don't think someone will think he/she will need to use all the options below to be able to chat with someone else, it doesn't make sense.

Reply:
> I don't think someone will think he/she will need to use all the options below to be able to chat with someone else, it doesn't make sense.

OK. fixed as what you want.